src/app/stores/UserStore.ts

```diff
@@ -60,6 +60,7 @@ export class UserStore {
         runInAction(() => {
           this.portfolios = portfolios
           this.hasLoadedPortfolios = true
+          this.rootStore.ui.setActivePortfolio(portfolios[0].id)
```

Comment:
cia tas tai jauciu kiekviena karta kai updeitinsis portfelis permes i pirma?

Reply:
good point, is ryta sutvarkysu